save-common/src/commonMain/kotlin/com/saveourtool/save/core/plugin/Plugin.kt

```diff
@@ -42,9 +44,10 @@ abstract class Plugin(
     /**
      * Perform plugin's work.
      *
+     * @param evaluatedToolConfig a configuration for evaluated tool
      * @return a sequence of [TestResult]s for each group of test resources
      */
-    fun execute(): Sequence<TestResult> {
+    fun execute(evaluatedToolConfig: EvaluatedToolConfig): Sequence<TestResult> {
```

Comment:
I think `evaluatedToolConfig` can be made `Plugin`'s property, just like `testConfig`. Maybe, not now, but when (if) we introduce tool config files

Reply:
yes (probably), but now adding it to Plugin level requires more changes (it tried it in my first iteration )

Reply:
will be handled in scope of https://github.com/saveourtool/save-cli/issues/435